frontend/e2e-tests/dom-to-db-tests/data-entry.e2e.ts

```diff
@@ -448,7 +450,9 @@ test.describe("navigation", () => {
         total_votes_cast_count: "100",
       };
       await votersVotesPage.fillInPageAndClickNext(voters, votes);
-      await votersVotesPage.acceptWarnings.click();
+
+      // eslint-disable-next-line playwright/no-force-option -- force option needed to click on hidden element
+      await votersVotesPage.acceptWarnings.click({ force: true, noWaitAfter: true });
```

Comment:
Maybe we should also using be `.check` here? (cc @jschuurk-kr)

Reply:
That would be better. I'd also prefer to have a `checkAcceptWarnings()` method in the page object instead of having to do this everywhere we want to accept warnings.

More importantly, locally the test fails every time at this point for me. Adding a `await expect(votersVotesPage.warning).toBeVisible();` before accepting the warning solved that.

Reply:
I've added a "checkAcceptWarnings" function to VotersVotesPgObj.ts
interestingly, there doesnt need to be a lint disable warning line here.

Reply:
That's because the page object files are not included in the eslint `overrides` for Playwright.

Addressed in 772d6e770ea0bc382e08b31bb830e0e26d012c32.